handlers.c

```diff
@@ -745,17 +745,25 @@ bool handler__lregions(globals_t * vars, char **argv, unsigned argc)
     return true;
 }
 
+static bool do_search ( scan_match_type_t m, globals_t * vars, uservalue_t * pval, char ** argv, unsigned argc );
```

Comment:
could you please tell me which coding style should i use, so i can setup my uncrustify utility. thanks a lot